tests/behat/features/deputy/07-infra-checks/00-documents.feature

```diff
@@ -0,0 +1,120 @@
+Feature: Infrastructure document tests
+
+  @infra
+  Scenario: Can upload documents
+    Given I am logged in as "laydeputy103@publicguardian.gov.uk" with password "Abcd1234"
+    When I set the report start date to "1/1/2016"
+    And I set the report end date to "31/12/2016"
+    And I click on "report-start, edit-documents, start"
+    And I fill in "document_wishToProvideDocumentation_0" with "yes"
+    And I click on "save-and-continue"
+    And I attach the file "empty-file.pdf" to "report_document_upload_file"
```

Comment:
Should this be testing the file checker for an error?

Reply:
Good point, added a check now